src/themes/light.scss

```diff
@@ -3,7 +3,9 @@
   --wxc-text-color: #{$wxc-black-alpha-95};
   --wxc-meeting-border: 1px solid #{$wxc-black-alpha-20};
   --wxc-meeting-box-shadow: 0px 24px 48px #{$wxc-black-alpha-20}, 0px 0px 1px #{$wxc-black-alpha-12};
-  
+  --wxc-tab-background: #{$wxc-black-alpha-20};
```

Comment:
i think we need the following variables:
--wxc-tab--normal--color
--wxc-tab--normal--background
--wxc-tab--active--color
--wxc-tab--active--background

Please put them after --wxc-modal*, to keep alpha order